docs/reference/scripts.md

```diff
@@ -49,6 +49,17 @@ Lua `print` builtin function writes skipper info log messages.
 
 `sleep(number)` function pauses execution for at least `number` milliseconds. A negative or zero duration causes `sleep` to return immediately.
 
+## Enable and Disable lua sources
+
+The flag `-lua-sources` allows to set 5 different values:
+
+* "file": Allows to use reference to file for scripts
+* "inline": Allows to use inline scripts
+* "inline", "file": Allows to use reference to file and inline scripts
+* "none": Disable Lua filters
+* "": the same as "inline", "file", the default value for binary and
+  library users
```

Comment:
"inline" for binary and "file" for library?
In this case I think we need to add "respectively" to the end of this sentence.

Reply:
No both are for users that create routes (our endusers), but skipper binary or skipper library users are the ones running the system.